src/context/AudioContext.js

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+const defaultErrorThunk = () => async () => {
+  throw new Error(
```

Comment:
Actually, there are just we who use this. Isn't it a bit... over-engineered? We won't intentionally remove this.

But I don't have any strong opinion